highfidelity/hifi
# MS19550 - Don't show help for HMDs that aren't plugged in and active

This PR is related to [MS#19550](https://highfidelity.fogbugz.com/f/cases/19550/Fix-VRC-PC-Functional-8-Vive-assets-cannot-be-displayed-in-Oculus-Store-build). Ensures only relevant HMD is shown when help is opened.

Originally, this project called for checking against the --oculus-store flag and having a special case, but after discussion with Seth and Brad it was decided that we shouldn't worry about a Vive being plugged in from the Oculus store, we just needed the help menu behavior fixed. See Brad's comment below.

**Test Plan:**
Phase 1.)
- Fire up interface with Oculus plugged in and two touch controllers connected..
- Open the Controller Help menu (Help > Controls Reference)
- Verify that only the Oculus, KBM, and Gamepad help images show up when you scroll through the layouts.

Phase 2.)
- Connect an Oculus headset _without_ touch controllers (I pulled the batteries from mine for testing).
- Connect an Xbox controller (make sure to do this _before_ firing up interface).
- Fire up interface.
- Open the Controller Help menu (Help > Controls Reference)
- Verify that only the KBM and gamepad layouts are showing (it should open to the Gamepad by default).

Phase 3.)
- Fire up interface with Vive plugged in.
- Open the Controller Help menu (Help > Controls Reference)
- Verify that only the Vive, KBM, and Gamepad help images show up when you scroll through the layouts.

Phase 4.)
- Fire up interface with Windows MR plugged in.
- Open the Controller Help menu (Help > Controls Reference)
- Verify that only the Windows MR, KBM, and Gamepad help images show up when you scroll through the layouts.

Assuming Test Phase 1 - 4 pass, this PR is good to go.
